package.json

```diff
@@ -39,16 +39,20 @@
   },
   "devDependencies": {
     "@types/jest": "23.1.4",
+    "@typescript-eslint/eslint-plugin": "^4.8.2",
+    "@typescript-eslint/parser": "^4.8.2",
     "all-contributors-cli": "6.16.0",
     "concurrently": "4.1.0",
+    "eslint": "^7.14.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.5",
     "husky": "1.1.2",
     "jest": "23.3.0",
     "lerna": "3.13.1",
-    "prettier": "1.13.7",
-    "pretty-quick": "1.6.0",
+    "prettier": "^2.2.0",
+    "pretty-quick": "^3.1.0",
```

Comment:
FYI, we used to prefer fixed versions to avoid bad surprises. This is especially true since it was a side-projects and we may not touch the dependencies for years. It prevents the "you come back 2 months later and nothing works" because someone didn't follow SemVer.

Do you have a preference for caret versions? If so, I'd be curious to discuss. 

☝ That's not blocking though.

Reply:
> It prevents the "you come back 2 months later and nothing works" because someone didn't follow SemVer.

I can definitely relate to this on some level, but admittedly I've not experienced this problem in major ways in the past years. For reference, I maintain somewhat sizable libraries that have low-maintenance to them today:

- [PlopJS](https://github.com/plopjs/plop)
- [Node Vibrant](https://github.com/Vibrant-Colors/node-vibrant)

And have had a helping hand in even larger projects that have little maintenance to them nowadays and I've found that the offset of "hey, you need to bump your patch version to fix this audit issue" balanced out the few minor times that it takes to update a broken dep.

Even with libraries with extremely low-maintenance mode (like Node Vibrant, that I've effectively halted development for), I'm still more than willing to jump in if it's something as simple as locking a package bump for

That said, it's far from a hill I'm willing to die on - if you'd rather I switch to fixed versions I'm definitely up for it

Reply:
Ah that's fun, I was a contributor on PlopJS too: https://github.com/plopjs/plop/graphs/contributors

I see your point. I'm totally fine with it too and I think it's better you use the convention you're familiar with if you start maintaining it (again, I'm not so 🤷‍♂️ ) 👍